public/js/main.js

```diff
@@ -185,9 +185,34 @@ filters.addEventListener("click",(event) => {
   const filterSelected = document.getElementsByClassName("selected")[0];
   filterSelected.removeAttribute("class");
   element.className = "selected";
-  renderData();
+  todoListComputed();
 });
 
+listData.addEventListener("dblclick",(event) => {
+  const element = event.target;
+  const item = element.parentNode.parentNode;
+  item.classList.add("editing");
+  const inputEdit = item.lastElementChild;
+
+  inputEdit.onkeypress = (e) => {
```

Comment:
Why do we bind the event here?

- [ ] explain

Reply:
I want to get the key pressed and then compare it with "enterKey" to know when user presses enter

Reply:
The event is bound every time the `dblclick` is fired. Can we bind this event when the input is initialized?

- [ ] consider